internal/server/handler_v2.go

```diff
@@ -18,14 +18,57 @@ package server
 import (
 	"context"
 
+	v2 "github.com/datacommonsorg/mixer/internal/server/v2"
 	v2pv "github.com/datacommonsorg/mixer/internal/server/v2/propertyvalues"
+	"github.com/datacommonsorg/mixer/internal/util"
 
 	pb "github.com/datacommonsorg/mixer/internal/proto"
 )
 
-// PropertyValuesV2 implements API for mixer.PropertyValuesV2.
-func (s *Server) PropertyValuesV2(
-	ctx context.Context, in *pb.PropertyValuesV2Request,
-) (*pb.PropertyValuesV2Response, error) {
-	return v2pv.API(ctx, in, s.store)
+// QueryV2 implements API for mixer.QueryV2.
+func (s *Server) QueryV2(
+	ctx context.Context, in *pb.QueryV2Request,
+) (*pb.QueryV2Response, error) {
+	arcStrings, err := v2.SplitArc(in.Graph)
```

Comment:
Style thing: always prefer using Get function, to avoid nil pointer..., so:

in.GetGraph()